Tools/autotest/quadplane.py

```diff
@@ -1513,6 +1513,41 @@ def Q_GUIDED_MODE(self):
 
         self.fly_home_land_and_disarm()
 
+    def DCMClimbRate(self):
+        '''Test the climb rate measurement in DCM with and without GPS'''
+        self.wait_ready_to_arm()
+
+        self.change_mode('QHOVER')
+        self.arm_vehicle()
+        self.set_rc(3, 2000)
+        self.wait_altitude(30, 50, relative=True)
+
+        # Start Descending
+        self.set_rc(3, 1000)
+        self.wait_climbrate(-5, -0.5, timeout=10)
+
+        # Switch to DCM
+        self.set_parameter('AHRS_EKF_TYPE', 0)
+        self.delay_sim_time(5)
```

Comment:
Would be better to sense the change in state here rather than a simple delay.  These types of sleeps lead to race conditions.